apps/custom-ui-floating-window-demo/components/Dialer/ActionButtons.js

```diff
@@ -0,0 +1,142 @@
+const ActionButtons = (props) => {
+  // Takes the user back to dialer by changing the state
+  let hangup = async () => {
```

Comment:
Can you add one more eslint rule to prioritise `const` over `let` - https://eslint.org/docs/latest/rules/prefer-const
`let` should be only used if you plan to change the variable value, but in this case it's never reassigned

Reply:
Sure. added this :)